designation_errors.md

```diff
@@ -17,3 +17,19 @@ HTTP/1.1 422 Unprocessable Entity
 ```
 
 The request body did not match the expected request schema.  Please check your parameters and try again.
+
+```
+HTTP/1.1 422 Unprocessable Entity
+```
+
+```json
+{
+  "errors": {
+    "decision_id": [
+      "Cannot set the decision_id when the local_status_id is not null"
```

Comment:
Should this be `local_status_id` or `local_status`?  (The API doesn't expose anything called `local_status_id`.)